jsons/Units.json

```diff
@@ -111,8 +111,8 @@
 		"requiredTech": "Metallurgy",
 		"obsoleteTech": "Combined Arms",
 		"upgradesTo": "Anti-Tank Gun",
-		"uniques": ["Can move after attacking","No defensive terrain bonus",
-			"[-33]% Strength <vs cities>","Consumes [1] [Horses]"],
+		"requiredResource": "Horses",

```

Comment:
requiredResource is an old way of doing it.

Reply:
it is old, but still working.

Reply:
Can you show me something that says the new way is to add it as unique?

Reply:
https://github.com/yairm210/Unciv/pull/5920

Reply:
Uniques are also used - and the only error i was having had to do with Venice ability.

requiredResource etc. will be obsolete sooner or later.

Reply:
You can use uniques, but requiredResource etc. are not (yet) outdated, with (as far I've seen) no set date when they will become outdated (if at all), and until that point it's (in my opinion) plain ugly to read "Consumes 1 Horses" in the description of a Cavalry-replacement that self-explanatory consumes horses (if it doesn't, it is shown as "Horses not required"). That description is entirely redundant. Do you find it to be visually appealing? (if that's true, then you of course must not merge those changes)